src/main/java/com/google/idea/perf/TracerController.kt

```diff
@@ -67,9 +72,15 @@ class TracerController(
     private var callTree = MutableCallTree(Tracepoint.ROOT)
     private val dataRefreshLoopStarted = AtomicBoolean()
 
+    private val autocomplete = Autocomplete(AgentLoader.instrumentation!!.allLoadedClasses.filter {
```

Comment:
If the instrumentation agent fails to load, then `AgentLoader.instrumentation!!` will throw an NPE. Let's try to degrade more gracefully in that case. (Eventually it'd be nice to show the user a warning message, but in the meantime let's at least avoid the NPE.)

Separately: `allLoadedClasses` will change over time, so we'll probably want to recompute it occasionally. (Although not too frequently---in my experience calling `allLoadedClasses` takes somewhere around 200 ms).